packages/dropdowns/src/elements/Multiselect/Multiselect.tsx

```diff
@@ -269,16 +304,20 @@ const Multiselect = React.forwardRef<HTMLDivElement, IMultiselectProps & ThemePr
       props.isCompact
     ]);
 
+    const isContainerHovered = isLabelHovered && !isOpen;
+    const isContainerFocused = isOpen ? true : isFocused;
```

Comment:
Not a request for change, but if the earlier concise substitute is safe (and preferred), then here's another location where  `isOpen || isFocused` could be used.